mycroft/audio/service.py

```diff
@@ -75,7 +78,91 @@ def __init__(self, ready_hook=on_ready, error_hook=on_error,
             LOG.exception(e)
             self.status.set_error(e)
 
+    @staticmethod
+    def get_tts_lang_options(lang, blacklist=None):
```

Comment:
Should include a docstring to specify return value structure and expected `lang` (BCP-47?)